app/Http/Requests/Settings/UpdateRequest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Http\Requests\Settings;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+          'projects_submission_start_at' => 'date',
+          'projects_submission_end_at' => 'required|date',
+          'min_members_team' => 'required|numeric|min:1|max_field:max_members_team',
+          'max_members_team' => 'required|numeric|min_field:min_members_team',
+        ];
+    }
+
+    /**
+     * Get custom attributes for validator errors.
+     *
+     * @return array
+     */
+    public function attributes()
+    {
+        return [
+            'projects_submission_start_at' => 'begin date of the submitting applications period',
+            'projects_submission_end_at' => 'end date of the submitting applications period',
+            'min_members_team' => 'min number of members per team',
+            'max_members_team' => 'max number of members per team',
+        ];
+    }
+
+    /**
+     * Get custom messages for validator errors.
+     *
+     * @return array
+     */
+    public function messages()
+    {
+        return [
+            'min_members_team.*' => 'The minimum number of members must between 0 and the maximum number of members.',
```

Comment:
I was not override, I was just trying to add some more info, but I will remove this.

Reply:
I will let the `min` and `max` stay, otherwise because they are custom extension rules the result would be something like this: 
![screenshot from 2018-01-11 16-49-00](https://user-images.githubusercontent.com/17494742/34836367-a71d4814-f6ef-11e7-97de-9a771582883f.png)

Reply:
This is a global rule and you will need this on every request that uses it, so add it to the file `validation.php`.